repokid/cli/repokid_cli.py

```diff
@@ -217,8 +217,8 @@ def _get_aardvark_data(aardvark_api_location, account_number=None, arn=None):
     while True:
         params = {'count': PAGE_SIZE, 'page': page_num}
         try:
-            r_aardvark = requests.post(aardvark_api_location, params=params, json=payload)
-        except requests.exceptions.RequestException as e:
+            r_aardvark = aardvark_service.post(aardvark_api_location, params=params, json=payload)
+        except BadResponse as e:
```

Comment:
Should we catch `BadResponse in the `aardvark_service` and throw a specialized exception? Reason for this is that `BadResponse` is Lyft specific, right?